gnocchi/carbonara.py

```diff
@@ -569,9 +577,6 @@ def transform(self, transform):
                 ts = ts.resample(sampling)
                 timestamps = ts["timestamps"]
                 values = ts["values"]
-
-            else:
-                raise ValueError("Transformation '%s' doesn't exists" % trans)
```

Comment:
How is it that you can remove that? I miss something.

Reply:
it never gets to this code which is why the tests never saw the error uncaught. when the api validates the transformation, it already gives a list of known methods: https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/rest/transformation.py#L54

Reply:
ok-ish. I still like to think this is an independent layer that can be used elsewhere so I like having that here.